config/database.yml

```diff
@@ -11,7 +11,7 @@ default: &default
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: /tmp/db/development.sqlite3
```

Comment:
we had a problem with the database as we needed to do something with it (don't know if we wanted to migrate or to drop it anymore) because of it was running in vagrant and this was the workaround
this change it should probably not be merged into the dev

Reply:
Fixed as of 797cff1a2bfd56453d10fc8d3ac9df75ad5ae1bd